src/main/scala/org/broadinstitute/dsde/vault/BossClientService.scala

```diff
@@ -20,10 +20,10 @@ object BossClientService {
   case class BossResolveObject(obj: BossResolutionRequest, id: String, creationKey: String)
   case class BossObjectResolved(bossObject: BossResolutionResponse, creationKey: String)
 
-  def props(requestContext: RequestContext): Props = Props(new BossClientService(requestContext))
+  def props(requestContext: RequestContext, version: Int): Props = Props(new BossClientService(requestContext, version))
 }
 
-case class BossClientService(requestContext: RequestContext) extends Actor {
+case class BossClientService(requestContext: RequestContext, version: Int) extends Actor {
```

Comment:
I don't think it makes sense to pass these versions to BOSS and DM client actors.  The DM behavior required by v3 of UBam Ingest has no connection to what v3 of Analysis Describe needs.